docs/deployment/allowed-registries.md

```diff
@@ -18,8 +18,8 @@ Re-tag the image and upload it to Docker Package Registry:
 
 ```bash
 docker pull unlisted/image:tag
-docker tag unlisted/image:tag docker.pkg.github.com/navikt/reponame/imagename:tag
-docker push docker.pkg.github.com/navikt/reponame/imagename:tag
+docker tag unlisted/image:tag ghcr.io/navikt/reponame/imagename:tag
```

Comment:
the format here should be `ghcr.io/org/imagename:tag`. Maybe review changes in other repos if one was unaware of this.

Reply:
Fair enough - I just went for "minimal change" =)

Good point!

Reply:
not 100% sure, but don't think it would work